tasks/section5.yml

```diff
@@ -237,6 +242,8 @@
       line: 'HostbasedAuthentication no'
   when:
       - rhel7cis_rule_5_2_7|bool
+      - rhel7cis_rule_5_2_7
```

Comment:
Why have the condition twice?

Reply:
Thanks, updated with DRY in commit message.